docs/ota-client-guide/modules/ROOT/pages/evaluation-to-prod.adoc

```diff
@@ -8,19 +8,26 @@ We recommend that you link:https://docs.ota.here.com/ota-client/latest/{docname}
 endif::[]
 
 
-The procedures to deploy OTA Connect in production are a little more complex than the basic xref:getstarted::index.adoc[Get Started]  procedures.
+The procedures to deploy OTA Connect in a realistic production scenario are a little more complex than the basic xref:getstarted::index.adoc[Get Started] procedures. But, you don't need to commit the resources for a full production roll-out right at the beginning of the process.
 
-That's why it's better work in phases when setting up your OTA Connect in your organization. We recommend that you work in three main phases:
+That's why it's better work in phases when setting up OTA Connect in your organization. We recommend that you work in three main phases:
 
 . __Evaluate OTA Connect__
-. __Build your own OTA-enabled solution__
-. __Deploy your own OTA-enabled solution__
+. __Build a realistic OTA-enabled PoC__
+. __Deploy a fully production-ready OTA solution__
+
+No matter which phase you're in, there are 4 basic tasks you'll need to do:
+
+* Build device images with an aktualizr-based client on it
+* Sign and upload the device images
+* Provision devices with authentication credentials for your account
+* Send updated images to the right devices
```

Comment:
"to the right devices" sounds funny to me. I get what you're trying to say, but would "to your devices as desired" by better?

Reply:
changed to "to some or all of your devices".